Fix:

warning: implicit conversion shortens 64-bit value into a 32-bit value

https://github.com/BlueM/cliclick/pull/111
--- Actions/MouseBaseAction.m.orig	2020-10-01 16:34:06.000000000 -0500
+++ Actions/MouseBaseAction.m	2020-10-01 16:41:13.000000000 -0500
@@ -164,3 +164,3 @@
     float yDiff = (endY - startY);
-    float stepSize = 1.0 / (float)steps;
+    float stepSize = (float)1.0 / (float)steps;
 
@@ -179,3 +179,3 @@
           dY = a.y - b.y;
-    return sqrt(dX * dX + dY * dY);
+    return sqrtf(dX * dX + dY * dY);
 }
@@ -196,3 +196,3 @@
         float f = ((2 * p) - 2);
-        return 0.5 * f * f * f + 1;
+        return (float)0.5 * f * f * f + 1;
     }

